misc/assets/obligees.svg

```diff
@@ -34,15 +34,15 @@
       <rect class="box" x="-100" y="-20" width="200" height="40" fill="none" stroke="black" stroke-width="1pt" stroke-dasharray="3pt" />
     </g>
   </defs>
-  
-  <g id="message" transform="translate(300 200)">
+
+  <g transform="translate(300 100)">
     <use href="#model" />
     <text class="caption">Obligee</text>
   </g>
-  <g transform="translate(700 200)">
+  <g transform="translate(700 100)">
     <use href="#external" />
     <text class="caption">Historical Obligee</text>
     <path class="relation from-many to-one" d="M -100,0 -300,0" />
-    <text class="note" x="-120" y="20">1+</text>
+    <text class="note" x="-120" y="0">1+</text>
```

Comment:
@viliambalaz Class `note` ma `dominant-baseline: ideographic;` co zabespeci, ze text sa automaticky vertikalne umiestni tesne nad y poziciu. Vid ako je napriklad text "Polymorphic relation" v `attachments.svg`.

To ci je text "1+" nad alebo pod ciarou je jedno. A radsej to umiestnit implicitne, nech sa vziadelosti od ciary vypocitaju automaticky, nez na konkretne pixely.

Ak by sme niekedy v buducnosti predsa len chceli dat text pod ciaru, tak tiez radsej namiesto explicineho zadania pixelov pouzije pre `y` suradnice ciary a nastavme spravne `dominant-baseline`.